R/workflow.R

```diff
@@ -19,11 +19,11 @@ Workflow <- R6::R6Class(
     #' @return A new `Workflow` object
     initialize = function(simulationSets,
                           workflowFolder) {
-      
+
       private$.reportingEngineInfo <- ReportingEngineInfo$new()
       # Check workflow folder input:
       # If workflow folder already exist throw a warning indicating the folder can be overwritten
-      self$workflowFolder <- workflowFolder %||% getwd()
+      self$workflowFolder <- workflowFolder %||% getwd()   #DO WE NEED THE %||% getwd() if workflowFolder is a required input?

```

Comment:
Agreed. Not required. Please remove
However VALIDATE that the workflowFolder is defined in the method using validateIsString() or sthg like this

Reply:
@abdullahhamadeh This needs to be updated

Reply:
done